eway/rapid/exception.py

```diff
@@ -102,6 +103,44 @@ def lookup_error_by_code(code, *args, **kwargs):
         if system_error:
             return system_error
 
+        undocummented_error = UndocummentedError.from_code(code)
```

Comment:
undocumented usually comes with a single `m` :)

Reply:
Woops. Thanks for catching this one.